### PR TITLE
Add option to convert more file at the same time

### PR DESCRIPTION
**Pull request by nicklayb**
I thought it could be great if your package could convert more than 1 file at the time.

I just itterated through all the given files.

Hope you like it!

### CODE REVIEW

**Comment by swrobel** [comment]
Thanks! This is great, but I have two suggestions:

1. Multiple input files is incompatible with the `-o`/`--outputfile` option, since it would just keep overwriting that same outputfile over and over again. Please add a check that ignores that option if you are processing multiple input files and notifies the user that it was ignored.
2. Please update the `README.md`

**Comment by swrobel** [comment]
Sorry, I should've been more clear. The changes all look good, please just update `README.md` to indicate that it is possible to pass multiple input files. An example would also be nice.

**Comment by nicklayb** [comment]
I ignored the option -o, but I wondered if you would prefer to cancel the conversion if the user provides -o options with more than one file?

**Comment by nicklayb** [comment]
Is it good for you?

File: README.md [hunk 1]
```diff
@@ -9,8 +9,14 @@ Sorry if the name causes any confusion but this gem doesn't currently output Sas
     % gem install less2sass
 
 ### Usage
+#### Single file conversion
+
     % less2sass inputfile.less [options]
 
+#### Multiple file conversion
+
+    % less2sass *.scss
```
Comment (swrobel) [inline README.md]: Please change to `% less2sass *.less [options]`. Also, add the following example `% less2sass inputfile1.less inputfile2.less [options]`

**Comment by swrobel** [comment]
Thanks! I'll put together a new release as soon as possible.

**Comment by swrobel** [comment]
1.2.0 is released!

**Pull request merged**